.ci/component_descriptor

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+set -e
+echo "enriching creating component descriptor from ${BASE_DEFINITION_PATH}"
+${ADD_DEPENDENCIES_CMD} --component-dependencies '{"name": "github.com/gardener/gardenctl", "version": "${cat GARDENCTL_VERSION}" }'
```

Comment:
I think there are a few syntax issues with`${cat GARDENCTL_VERSION}`:
- To execute cat in a subshell, the syntax is `$()` instead of `${}`
- The single quotes (`'`) will not allow this to expand and will treat the cat literally, the simplest option would be to use double quotes througout (`"`) and escape the internal double quotes (`\"`)